src/Microsoft.VisualStudio.Editor.Razor/DefaultVisualStudioRazorParser.cs

```diff
@@ -110,7 +102,7 @@ internal DefaultVisualStudioRazorParser(RazorCodeDocument codeDocument)
         // Used in unit tests to ensure we can block background idle work.
         internal ManualResetEventSlim BlockBackgroundIdleWork { get; set; }
 
-        public override async Task ReparseAsync()
+        public override async void QueueReparse()
         {
             // Can be called from any thread
```

Comment:
You can safely remove the 'await' from this method. There's no need to await that task